time time
string name
string emotion